pagopa/io-app
# chore: [IODPAY-207] Move initiatives from instrument refresh logic in saga

## Short description
moved initiatives from instrument refresh logic from a custom hook to redux-saga

## List of changes proposed in this pull request
moved initiatives from instrument refresh logic from a custom hook to redux-saga.
This was achieved with a subscribable channel that allows us to handle multiple refresh sagas one-by-one, thus putting aside inconsistent action dispatch timings from react-native's stackNavigator.

## How to test
1. run the test suite
2. navigate to a payment method's details screen and make sure the initiatives section renders correctly
